Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,60 @@
+class Tarantool < Formula
+  def instances_enabled
+    etc+"tarantool/instances.enabled"
+  end
+
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-562-g3dbc9ff-src.tar.gz"
+  version "1.6.7-562"
+  sha256 "6aded257e11fa114dc21e8493ff96dfe3f8a692d62e096afb5105cecc179211d"
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
+    version "1.6.8"
+  end
+
+  head do
+    url "https://github.com/tarantool/tarantool.git", :branch => "master", :shallow => false
+  end
+
+  depends_on "cmake" => :build
+  depends_on "readline" => :build
+
+  def install
+    args = []
+
+    args << "-DCMAKE_INSTALL_MANDIR=#{doc}"
+    args << "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}"
+    args += std_cmake_args
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+    inreplace bin/"tarantoolctl" do |s|
+      s.gsub!(%r{\/usr\/local\/etc\/tarantool\/default\/tarantool},
+              "#{etc}/default/tarantool")
+    end
+
+    inreplace etc/"default/tarantool" do |s|
+      local_user = ENV["USER"]
+      s.gsub!(/(pid_file\s*=).*/, "\\1 '#{var}/run/tarantool',")
+      s.gsub!(/(wal_dir\s*=).*/, "\\1 '#{var}/lib/tarantool',")
+      s.gsub!(/(snap_dir\s*=).*/, "\\1 '#{var}/lib/tarantool',")
+      s.gsub!(/(sophia_dir\s*=).*/, "\\1 '#{var}/lib/tarantool',")
+      s.gsub!(/(logger\s*=).*/, "\\1 '#{var}/log/tarantool',")
+      s.gsub!(/(username\s*=).*/, "\\1 '#{local_user}'")
+      s.gsub!(/(instance_dir\s*=).*/, "\\1 '#{etc}/tarantool/instances.enabled'")
+    end
+
+    mkdir_p "#{var}/run/tarantool"
+    mkdir_p "#{var}/lib/tarantool"
+    mkdir_p "#{var}/log/tarantool"
+    mkdir_p instances_enabled
+    instances_enabled.install_symlink "#{etc}/tarantool/instances.available/example.lua"
+  end
+
+  test do
+    system bin/"tarantool", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!